Procfile

```diff
@@ -1,2 +1,2 @@
 web: bin/rails server -p $PORT -e $RAILS_ENV
-release: bin/rails db:migrate && bin/rails db:seed && bin/rails r -e production lib/production_tasks.rb
+release: bin/rails db:migrate && bin/rails db:seed
```

Comment:
Ja bym usunal db:seed - seedy odpalane na serwerze sa tak rzadko, ze lepiej to zrobic "z palca"